libs/community/langchain_community/utilities/__init__.py

```diff
@@ -302,8 +297,19 @@
     "ZapierNLAWrapper": "langchain_community.utilities.zapier",
 }
 
+DEPRECATED = {
```

Comment:
```suggestion
REMOVED = {
```
maybe rename to removed just to be clear